docs/_documentations/codewind-openapi-vscode.md

```diff
@@ -22,6 +22,21 @@ The Codewind OpenAPI Tools for VS Code provides commands that invoke the OpenAPI
 3. This extension pulls the [OpenAPI Generator CLI Docker Image](https://github.com/OpenAPITools/openapi-generator#16---docker) and runs the OpenAPI Generator in a Docker container. Install Docker if necessary.
 4. Install the Codewind OpenAPI Tools from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-openapi-tools) or by searching for `Codewind OpenAPI` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
 
+## Generating and building code in an existing Java Maven project
+1. Right-click your Codewind project and launch the **Generate Server from OpenAPI Definition** wizard.
+2. Select your OpenAPI document `.json` file.
+3. Select the `jaxrs-spec` server generator type.
+4. When you're prompted to select the output folder, keep the default selection. The project root, which is the parent of the source folder, is selected by default.
+5. Click **Select As Output Folder** to merge the generated `.pom` file with the existing `.pom` file. The merge attempts to include all necessary dependencies for the project to build properly.
+6. When prompted with the warning message, note that the `pom.xml` file is backed up. However, XML comments, such as `<!-- example comment -->`, are removed. Click **Yes** to proceed. The files build.
```

Comment:
6. At this step, the currently supported generator options have been collected. Code generation is ready to begin until after a warning message to proceed is confirmed.  The existing `pom.xml` file is backed up first as `pom-backup.xml` or variation.  The new `pom.xml` will be a merged version of the original POM and the one that is generated.  The merge attempts to include all necessary dependencies for the project to build properly.  Due to limitations in the merge process, XML comments such as `<!-- example comment -->`, from the original POM are not preserved.   When prompted with the warning message, click **Yes** to proceed with code generation.  For any Codewind project, if autobuild is turned on, the project will build automatically.